integrations/uphold/src/main/java/org/dash/wallet/integrations/uphold/data/UpholdAddress.kt

```diff
@@ -14,28 +14,9 @@
  * You should have received a copy of the GNU General Public License
  * along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
+package org.dash.wallet.integrations.uphold.data
 
-package org.dash.wallet.integrations.uphold.data;
-
-public class UpholdAddress {
-
-    private String line1;
-    private String line2;
-
-    public String getLine1() {
-        return line1;
-    }
-
-    public void setLine1(String line1) {
-        this.line1 = line1;
-    }
-
-    public String getLine2() {
-        return line2;
-    }
-
-    public void setLine2(String line2) {
-        this.line2 = line2;
-    }
-
-}
+data class UpholdAddress(
+    val format: String,
+    val value: String
+)
```

Comment:
This class was never used previously for its contents.  The original code didn't know what line one and two were.  Now we know what they are and will use them.